models/users.js

```diff
@@ -188,6 +189,39 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ * Adds default archived role
+ * @return {Promise<usersMigrated|Object>}
+ */
+
+const addDefaultColors = async () => {
+  try {
+    const usersSnapshot = await userModel.get();
+    const migratedUsers = [];
+    const updateUserPromises = [];
+    const usersArr = [];
+
+    usersSnapshot.forEach((doc) => usersArr.push({ id: doc.id, ...doc.data() }));
+
+    for (const user of usersArr) {
+      const colors = user.colors ? user.colors : {};
+      if (user.colors === undefined) {
+        const userColorIndex = addColorsProperty(cardColorArray);
+
+        colors.primary_color = cardColorArray[parseInt(userColorIndex)].color_primary;
+        colors.secondary_color = cardColorArray[parseInt(userColorIndex)].color_secondary;
```

Comment:
Instead of setting color code directly here, can we store id that relates to that color.

Eg - In constant file we can have -
```
module.exports = [
  { color_id: 1, color_primary: "#DB1212", color_secondary: "#F88181" },
   ...
 ```
 and do `colors.colorId = randomColor.color_id`

Reply:
i got the part where we use id, but
`colors.colorId = randomColor.color_id`
would just add a property to colors like this:
`{colorId: 1}`

Reply:
Yes that's what we want. So in frontend, we assign colors to `colorId`. 

Let's say we want to change users with color red to blue. By the original approach, we need to update color of every user with color red. Using color_id we can change color in one line in frontend.

Reply:
oh, ok I get it, so does it make sense to mantain a colorArray at the backend as well. I think we would have to mantain the array on both FE and BE with this approach, wdyt?

Reply:
Ideally this should be stored in a remote config. Eg - Firebase remote config.

But for now we can use keep it in frontend only, if needed make a api, `/colors` for the color array.

Reply:
I have made the change, Can we approve the PR?